src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -47,6 +48,10 @@ function ResultsView(props: ResultsViewProps) {
   const { dispatchFetchCompareResults } = useFetchCompareResults();
   const { searchByRevisionOrEmail } = useHandleChangeSearch();
 
+  useEffect(() => {
+    document.title = props.title;
```

Comment:
you can use `title` directly here, this would be more correct too because that's the variable in the dependency below

Reply:
The same comment applies below